portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -68,8 +69,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     for (Entity entity : results.asIterable()) {
       long id = entity.getKey().getId();
       String content = (String) entity.getProperty("content");
+      String score = (String) entity.getProperty("sentimentScore");
+      int scoreInt = Integer.parseInt(score);
```

Comment:
You can merge this into Integer.parseInt( (String) entity.getProperty("sentimentScore")) or Integer.parseInt(entity.getProperty("sentimentScore".toString)

And scoreInt -> sentimentScore, to make it clearer what it is scoriong and to remove the extra type annotation

Reply:
Also, should this be a double? https://cloud.google.com/natural-language/docs/sentiment-tutorial#parsing_the_response